sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsClient.java

```diff
@@ -51,35 +51,27 @@
      * Returns the detected language and a numeric score between zero and one. Scores close to one indicate 100%
      * certainty that the identified language is true.
      *
-     * @param text The text to be analyzed.
-     * @return the {@link DetectLanguageResult detected language} of the text.
-     * @throws NullPointerException if {@code text} is {@code null}.
-     */
-    @ServiceMethod(returns = ReturnType.SINGLE)
-    public DetectLanguageResult detectLanguage(String text) {
-        return detectLanguage(text, client.getDefaultCountryHint());
-    }
-
-    /**
-     * Returns the detected language and a numeric score between zero and one when the hint of country specified.
-     * Scores close to one indicate 100% certainty that the identified language is true.
+     * <p><strong>Code Samples</strong></p>
+     * <p>Detects the languages of single input text</p>
+     * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsClient.detectLanguage#String}
      *
      * @param text The text to be analyzed.
-     * @param countryHint Accepts two letter country codes specified by ISO 3166-1 alpha-2. Defaults to "US" if not
-     * specified.
-     *
      * @return the {@link DetectLanguageResult detected language} of the text.
      * @throws NullPointerException if {@code text} is {@code null}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public DetectLanguageResult detectLanguage(String text, String countryHint) {
```

Comment:
why is the method getting changed here?

Reply:
I think this API is not required. It only takes input text and country hint. If you take a look at other APIs. You will find this API is an extra API. Either of AsyncClient doesn't have it or recognizeEntiteis, PII entities, LInked Entities, Extract KeyPhrases and Analyze Sentiments don't have it either.